index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en-us">
+  <head>
+    <meta charset="utf-8" />
+    <title>Sway Finance</title>
+    <link href="styles.css" rel="stylesheet" />
+  </head>
+  <body>
+    <img class="branding" src="images/user-can-see-branding-and-navigation.png" alt="Picture of logo and navigation options on top of webpage" />
```

Comment:
you can't just put the picture here. You have to re-create it using HTML and CSS.